opengever/maintenance/scripts/bumblebee_installation.py

```diff
@@ -111,7 +111,14 @@ def main(app, argv=sys.argv[1:]):
 
     if mode == 'reindex':
         LOG.info("Start indexing objects...")
-        converter.reindex()
+        try:
+            converter.reindex(intermediate_commit=options.intermediate_commit)
+        except TypeError:
+            if options.intermediate_commit:
```

Comment:
Should probably remove the `"Only has an effect on '-m history' at the moment."` from the options parser